test/unit/helpers/artefacts_helper_test.rb

```diff
@@ -13,4 +13,46 @@ class ArtefactsHelperTest < ActiveSupport::TestCase
       assert manageable_formats.exclude?('specialist_sector')
     end
   end
+
+  context "#action_information_phrase" do
+    setup do
+      @artefact = FactoryGirl.create(:artefact)
+    end
+
+    context "when action is performed by user" do
+      should "return user name and email" do
+        user = FactoryGirl.create(:user)
+        @artefact.actions.build(user: user, action_type: 'update')
+
+        expected = "#{user} <#{user.email}> has updated this artefact"
+        assert_equal expected, action_information_phrase(@artefact.actions.last)
+      end
+    end
+
+    context "when action is performed by task" do
+      should "return task name" do
+        @artefact.actions.build(task_performed_by: 'TaggingUpdater', action_type: 'update')
+
+        expected = "Automatic task: 'TaggingUpdater' has updated this artefact"
```

Comment:
Maybe for TaggingUpdater: `An external application has updated the tags for this artefact.`

For all else: `A developer has manually updated this artefact`.